src/components/StateBar/index.tsx

```diff
@@ -53,12 +53,15 @@ export const StateBarBar = ({
   value = 0,
   ...props
 }: StateBarBarProps): JSX.Element => {
+  const theme = useTheme()
+
   const variant = useMemo(() => {
     if (unlimited) return 'success'
```

Comment:
we can now using the new colors everywhere ? this will avoid misunderstood where the colors come from ?

Reply:
```js  
if (unlimited) return 'success'
```

will be 
```js
  if (unlimited) return theme.colors.therightcolorstobeused ?
```

Reply:
Good point @philibea, there is an issue that I didn't understand `theme.colors.succes.background` or whatever color from the new colors don't work, don't work I mean the bar show without color on storybook like the photo above

Reply:
@philibea ProgressBar expect a variant, not a color :)

Reply:
@chambo-e  do you think that the history of variant need to be changed to accept color as well as variant?

Reply:
@Lawndlwd nope, a variant is a variant, if we start accepting colors we are going to be lost in no time 😄